classes/Exceptions/UpgradeException.php

```diff
@@ -25,7 +25,7 @@
  * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License 3.0 (AFL-3.0)
  */
 
-namespace PrestaShop\Module\AutoUpgrade;
```

Comment:
On #715, the new namespace is `Exception` in the singular form

Reply:
Fixed with https://github.com/PrestaShop/autoupgrade/pull/715/commits/813724a52a9b942bc017e1f8e550f32b83a3982f